index.bs

```diff
@@ -285,6 +287,18 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 Perform Logger("warn", <var>data</var>).
 
+<h4 id="dir" oldids="dir-data,dom-console-dir" method for="console">dir(<var>data</var>)</h4>
```

Comment:
Yes, exactly; they were IDs that used to exist for those methods in the document before some restructuring/renaming. It looks like they were introduced in https://github.com/whatwg/console/commit/64a3ff6df3456daa5cc50efe3cd8e7141d4ea402; previously each heading had two separate anchors (one for the h4 and one for the dfn), both of which had suboptimal IDs. That commit gave everything optimal IDs but then introduced oldids so as to not break any inbound links.